docs/local-gadget.md

```diff
@@ -5,41 +5,203 @@ description: >
   How to use the local-gadget tool.
 ---
 
-Inspektor Gadget can also be used without Kubernetes to trace containers with the `local-gadget` tool.
+Inspektor Gadget can also be used without Kubernetes to trace containers with
+the `local-gadget` tool.
+
+Currently, the `local-gadget` can trace containers managed by Docker regardless
+of whether they were created via Kubernetes or not. In addition, it will also
+use the CRI to trace containers managed by Containerd and CRI-O, meaning only
```

Comment:
`Containerd` -> `containerd` (as written in https://containerd.io/).

Reply:
Applied everywhere, thanks!